mapboxgl/viz.py

```diff
@@ -276,3 +276,33 @@ def add_unique_template_variables(self, options):
             clusterRadius=self.clusterRadius,
             clusterMaxZoom=self.clusterMaxZoom
         ))
+
+
+class ImageViz(MapViz):
+    """Create a image viz"""
+
+    def __init__(self,
+                 data,
+                 image,
```

Comment:
@vincentsarago could we simplify the syntax by just passing the pixels or URL for an `image` to the required `data` parameter, instead of adding a new parameter?